<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog
{
	background-color:#18243C;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="btnNum1">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnNum9">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnNum4">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QPushButton" name="addBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #1D2E48;
}</string>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="modulusBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0px;

}
QPushButton:pressed {
    background-color: #1D2E48;
}
</string>
     </property>
     <property name="text">
      <string>⁒</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="btnNum2">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QPushButton" name="btnNum7">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QLineEdit" name="lineEdit">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit
{
	background-color:#18243C;
	border:none;
	min-height:50px;
	color:#fff;
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="signBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #1D2E48;
}</string>
     </property>
     <property name="text">
      <string>±</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="divBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #1D2E48;
}</string>
     </property>
     <property name="text">
      <string>÷</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="btnNum5">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QPushButton" name="mulBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>42</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}

QPushButton:pressed {
    background-color: #1D2E48;
}</string>
     </property>
     <property name="text">
      <string>x</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnNum8">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QPushButton" name="equalBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #1D2E48;
}</string>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="clearBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>25</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #1D2E48;
}</string>
     </property>
     <property name="text">
      <string>AC</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="subBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	point-size:35;
	border :none;
	border-radius:30px;
	background-color:#2869C7;
	color:rgb(255,255,255);
	min-width:60px;
	max-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #1D2E48;
}
</string>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="btnNum3">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="btnNum6">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="undoBtn">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
     <property name="iconSize">
      <size>
       <width>26</width>
       <height>26</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QPushButton" name="btnNum0">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>35</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton
{	
	border :none;
	border-radius:30px;
	background-color:#1D2E48;
	color:rgb(255,255,255);
	min-width:60px;
	min-height:60px;
	max-height:60px;
	padding:0;

}
QPushButton:pressed {
    background-color: #2869C7;
}</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
